# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/src/libANGLE/Observer.cpp
# Commit: 3a91885d47e0
# Full Hash: 3a91885d47e0ef7fba9749562d52300eea67c651
# Author: Miko Mynttinen <mikokm@gmail.com>
# Date: 2019-11-11 21:52:52
# Regressor Bug: 1578576
# File Overlap Count: 4
# Description:
#   Bug 1578576 - Part 2: Update to ANGLE 3865 r=jgilbert
#   
#   Depends on D44561
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D44579
# ==============================================================================

diff -r 77cee77c0307 -r 3a91885d47e0 gfx/angle/checkout/src/libANGLE/Observer.cpp
--- a/gfx/angle/checkout/src/libANGLE/Observer.cpp	Fri Nov 08 16:41:42 2019 +0000
+++ b/gfx/angle/checkout/src/libANGLE/Observer.cpp	Fri Nov 08 17:06:00 2019 +0000
@@ -37,14 +37,14 @@
     return !mObservers.empty();
 }
 
-void Subject::onStateChange(const gl::Context *context, SubjectMessage message) const
+void Subject::onStateChange(SubjectMessage message) const
 {
     if (mObservers.empty())
         return;
 
     for (const ObserverBindingBase *binding : mObservers)
     {
-        binding->getObserver()->onSubjectStateChange(context, binding->getSubjectIndex(), message);
+        binding->getObserver()->onSubjectStateChange(binding->getSubjectIndex(), message);
     }
 }
 
@@ -89,9 +89,9 @@
     }
 }
 
-void ObserverBinding::onStateChange(const gl::Context *context, SubjectMessage message) const
+void ObserverBinding::onStateChange(SubjectMessage message) const
 {
-    getObserver()->onSubjectStateChange(context, getSubjectIndex(), message);
+    getObserver()->onSubjectStateChange(getSubjectIndex(), message);
 }
 
 void ObserverBinding::onSubjectReset()